index.html

```diff
@@ -36,56 +36,196 @@
+                    </svg> </p>
+                <ul>
+                    <li class="sidebar-item"> <a href="https://stackoverflow.com/collectives"
+                            class="text-with-icon icon-left"> <svg xmlns="http://www.w3.org/2000/svg"
+                                viewBox="0 0 576 512">
+                                <!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
+                                <path
+                                    d="M316.9 18C311.6 7 300.4 0 288.1 0s-23.4 7-28.8 18L195 150.3 51.4 171.5c-12 1.8-22 10.2-25.7 21.7s-.7 24.2 7.9 32.7L137.8 329 113.2 474.7c-2 12 3 24.2 12.9 31.3s23 8 33.8 2.3l128.3-68.5 128.3 68.5c10.8 5.7 23.9 4.9 33.8-2.3s14.9-19.3 12.9-31.3L438.5 329 542.7 225.9c8.6-8.5 11.7-21.2 7.9-32.7s-13.7-19.9-25.7-21.7L381.2 150.3 316.9 18z" />
+                            </svg> Explore Colectives </a> </li>
+                </ul>
+            </div>
 
-        <p> Teams </p>
-        <div class="teams-card">
-            <p> <strong> Stack Overflow for Teams </strong> - Start collaborating and sharing
-                organizational knowledge.
-            </p>
-            <img src="./assets/teams-illo-free-sidebar-promo.svg" alt="teams-logo">
-            <a class="btn btn-orange block align-text-center mb-02 font-12"
-                href="https://try.stackoverflow.co/why-teams/?utm_source=so-owned&utm_medium=side-bar&utm_campaign=campaign-38&utm_content=cta">
-                Create a free Team </a>
-            <a class="align-text-center block font-12" href="https://stackoverflow.co/teams/"> Why Teams? </a>
+            <p> Teams </p>
+            <div class="teams-card">
+                <p> <strong> Stack Overflow for Teams </strong> - Start collaborating and sharing
+                    organizational knowledge.
+                </p>
+                <img src="./assets/teams-illo-free-sidebar-promo.svg" alt="teams-logo">
+                <a class="btn btn-orange block align-text-center mb-02 font-12"
+                    href="https://try.stackoverflow.co/why-teams/?utm_source=so-owned&utm_medium=side-bar&utm_campaign=campaign-38&utm_content=cta">
+                    Create a free Team </a>
+                <a class="align-text-center block font-12" href="https://stackoverflow.co/teams/"> Why Teams? </a>
+            </div>
+        </nav>
 
-        </div>
-    </nav>
+        <main>
+            <div class="padding-02">
+                <div class="flex justify-between align-center">
+                    <h1> All Questions </h1>
+                    <a class="btn btn-blue align-center inline-block"
```

Comment:
I don't think you need the `inline-block` class here. Check that the UI looks the same if you remove it.